terraform/18f.gov.tf

```diff
@@ -141,7 +141,7 @@ resource "aws_route53_record" "18f_gov_accessibility_18f_gov_a" {
   }
 }
 
-resource "aws_route53_record" "18f_gov_ads_18f_gov_a" {
+resource "aws_route53_record" "18f_gov_agile-bpa-old__18f_gov_a" {
```

Comment:
I would leave this as "ads" just to avoid messing with something that already works and call the new one "agile-bpa." Sorry not to leave better comments earlier; I was on my phone.